piccolo_admin/endpoints.py

```diff
@@ -149,6 +151,14 @@ def __init__(
                                 )
                             ),
                         ),
+                        Mount(
+                            path="/sitename/",
+                            app=auth_middleware(
+                                Router(
+                                    [Route(path="/", endpoint=self.get_sitename)]
+                                )
+                            ),
+                        ),
```

Comment:
Rather than having a separate endpoint, we could return the `site_name` as part of the meta endpoint?